lib/events/dynamoevents/dynamoevents.go

```diff
@@ -204,6 +230,28 @@ func New(cfg Config) (*Log, error) {
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
+
+	// Enable continuous backups if requested.
+	if b.Config.EnableContinuousBackups {
```

Comment:
How can continuous backups and auto-scaling be disabled? Or teleport does not make an attempt to do that?
I'm asking since the switches were advertised as working both ways but seems like they're used only to turn settings on. I also read the previous comment about the choice of bool vs pointer to bool which would've solved it but it was explicitly chosen to favor simplicity which means just turning it on?

Reply:
Yeah, now it only supports turning on and configuring.